cmd/commands/config/command.go

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package config
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/mysteriumnetwork/node/cmd/commands/cli/clio"
+	"github.com/mysteriumnetwork/node/config"
+	"github.com/mysteriumnetwork/node/config/urfavecli/clicontext"
+
+	"github.com/urfave/cli/v2"
+)
+
+// CommandName is the name which is used to call this command
+const CommandName = "config"
+
+// NewCommand function creates license command.
+func NewCommand() *cli.Command {
+	cmd := &command{}
+	return &cli.Command{
+		Name:        CommandName,
+		Usage:       "Manage your node config",
+		Description: "Using config subcommands you can view and manage your current node config",
+		Before: func(ctx *cli.Context) error {
+			if err := clicontext.LoadUserConfigQuietly(ctx); err != nil {
+				return err
+			}
+
+			config.ParseFlagsServiceStart(ctx)
+			config.ParseFlagsServiceOpenvpn(ctx)
+			config.ParseFlagsServiceWireguard(ctx)
+			config.ParseFlagsServiceNoop(ctx)
+			config.ParseFlagsNode(ctx)
+			return nil
+		},
+		Subcommands: []*cli.Command{
+			{
+				Name:  "show",
+				Usage: "Submit a registration request",
```

Comment:
how's the registration going?

Reply:
copy pasting is the hardest part about programming